messages/dji_msgs/msg/Topics.msg

```diff
@@ -6,4 +6,9 @@ string CONTROLLER_VIBRATION_TOPIC = 'joy/set_feedback'
 string TTS_TOPIC = 'speak'
 
 
-
+string CAMERA_TOPIC = 'Quadrotor/gimbal_camera/image_raw'
```

Comment:
remove `Quadrotor/`

Reply:
yes. because the _will_ change, there might be mulitple, and not all of them will have the same name. thats why your node needs to be namespace-able. having a hardcoded, starting-at-root topic prevents this.